programs/drift/src/instructions/keeper.rs

```diff
@@ -426,6 +435,145 @@ pub fn handle_update_user_open_orders_count<'info>(ctx: Context<UpdateUserIdle>)
     Ok(())
 }
 
+pub fn place_swift_taker_order<'c: 'info, 'info>(
```

Comment:
nit: think it's easier to read code if this fn is under handle_place_swift_taker_order